build.proj

```diff
@@ -78,28 +78,6 @@
              ContinueOnError="ErrorAndContinue" />
   </Target>
 
-  <!-- TODO: Can we move this archive test build to BuildTools -->
-  <UsingTask TaskName="ZipFileCreateFromDependencyLists" Condition="'$(ArchiveTests)' == 'true'" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
```

Comment:
Was this moved to buildtools? We will still need some form of this since even though we now only have one test directory, we will need to zip it in order to move it to Helix.